src/stdlib/stdlib/libs/deploy.tact

```diff
@@ -1,25 +1,113 @@
+/// Global function. Available since Tact 1.6.1.
+///
+/// Queues an empty message to be sent with the `SendRemainingValue` mode to the destination `Address` `to`. This is the most gas-efficient way to send the remaining value from the incoming message to the given address.
+///
+/// Attempts to queue more than 255 messages throw an exception with exit code 33: `Action list is too long`.
+///
+/// ```tact
+/// fun example() {
+///     // Forward the remaining value back to the sender
+///     cashback(sender());
+/// }
+/// ```
+///
+/// See: https://docs.tact-lang.org/ref/stdlib-deploy/#cashback
+///
+asm fun cashback(to: Address) {
```

Comment:
Let's move`cashback` to `src/stdlib/stdlib/std/internal/send.tact`, since it can be used even if the user prefers to explicitly implement their own deploy logic